tests/general/LineCountBearTest.py

```diff
@@ -40,3 +45,53 @@ def test_exclude_blank_lines(self):
                                 severity=RESULT_SEVERITY.NORMAL,
                                 file='default')],
             filename='default')
+
+    def test_of_range_allowed_for_lines_per_file(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('min_lines_per_file', 5))
+        self.check_results(
+            self.uut, ['line 1', '', 'line 2', '', 'line 3', 'line 4'],
+            [Result.from_values('LineCountBear',
+                                'This file had 4 lines, which is 1 lines '
```

Comment:
hmm, this is a good demonstration why the message needs polishing ;)
1 line*s*

Reply:
@Makman2 Should I to correct only my tests (for min_lines_per_file)? Maybe should I also to correct tests with max_lines_per_file?

Reply:
Ah it's the same? Then keep it, we can file an issue afterwards to correct both messages and use a grammatical-proof message :)